dbt_semantic_interfaces/implementations/metric.py

```diff
@@ -179,6 +179,8 @@ class PydanticCumulativeTypeParams(HashableBaseModel):
 class PydanticMetricAggregationParams(HashableBaseModel):
     """Type params to provide context for metrics that are used as source nodes."""
 
+    model: str
```

Comment:
nit but let's call this `semantic_model`? they're essentially 1:1 but in SL world (and in the context of this Pydantic object) we don't ever really refer to these as just models.